src/test/java/bowling/domain/StateTest.java

```diff
@@ -0,0 +1,45 @@
+package bowling.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class StateTest {
+    @Test
+    @DisplayName(value = "스트라이크")
+    void strike() {
+        assertThat(State.of(10, true))
+                .isEqualTo(State.STRIKE);
+    }
+
+    @Test
+    @DisplayName(value = "스페어")
+    void spare() {
+        assertThat(State.of(10, false))
+                .isEqualTo(State.SPARE);
+    }
+
+    @Test
+    @DisplayName(value = "미스")
+    void miss() {
+        assertThat(State.of(1, false))
+                .isEqualTo(State.MISS);
+    }
+
+    @Test
+    @DisplayName(value = "거터")
+    void gutter() {
+        assertThat(State.of(0, false))
+                .isEqualTo(State.GUTTER);
+    }
+
+    @Test
+    @DisplayName(value = "그외")
```

Comment:
```suggestion
    @DisplayName("그외")
```

자그마한 팁이지만, DisplayName 에 value 생략하더라도, 어노테이션이 매우 잘 동작한답니다.

Reply:
저도 이걸 왜 매번 넣었는지 의문이 드네여...;;